packages/manager/src/lib/mockSlice.ts

```diff
@@ -0,0 +1,36 @@
+import { SharedSliceMock } from "@prismicio/mocks";
+
+import { SharedSliceContent } from "@prismicio/types-internal/lib/content";
+import { SliceDiff } from "@prismicio/types-internal/lib/customtypes/diff";
+import { SharedSlice } from "@prismicio/types-internal/lib/customtypes/widgets/slices";
+
+type mockSliceArgs = {
+	model: SharedSlice;
+	mocks?: SharedSliceContent[];
+	sliceDiff?: SliceDiff;
```

Comment:
Don't want to nitpick, but since we're already providing a (slice) model, (slice) mocks, maybe we can align `sliceDiff` with that and rename it to (slice) `diff`?

Up to you!